src/main/kotlin/dev/fakek/fakes/FakeCompany.kt

```diff
@@ -13,7 +13,7 @@ package dev.fakek.fakes
  * @param suffix
  * @param url random url of company website.
  */
-data class FakeCompany(
+open class FakeCompany(
```

Comment:
Making this open will break constructs provided by the `data` class which include things like `toString`, `equals`, and `copy`. These classes should remain data classes as it offers those additional benefits to consumers of this library.

Here's more info on data classes: https://kotlinlang.org/docs/reference/data-classes.html